docs/docs.md

```diff
@@ -5984,6 +5993,117 @@ Configuration for webhooks.
 
 
 
+<a name="spinnaker/extensibility.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## spinnaker/extensibility.proto
+
+
+
+<a name="proto.spinnaker.Extensibility"></a>
+
+### Extensibility
+Extensibility flags
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| plugins | [Extensibility.PluginsEntry](#proto.spinnaker.Extensibility.PluginsEntry) | repeated |  |
+| repositories | [Extensibility.RepositoriesEntry](#proto.spinnaker.Extensibility.RepositoriesEntry) | repeated |  |
+
+
+
+
+
+
+<a name="proto.spinnaker.Extensibility.PluginsEntry"></a>
+
+### Extensibility.PluginsEntry
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [Plugin](#proto.spinnaker.Plugin) |  |  |
+
+
+
+
+
+
+<a name="proto.spinnaker.Extensibility.RepositoriesEntry"></a>
+
+### Extensibility.RepositoriesEntry
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [Repository](#proto.spinnaker.Repository) |  |  |
+
+
+
+
+
+
+<a name="proto.spinnaker.Plugin"></a>
+
+### Plugin
+Spinnaker plugin flags
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| enabled | [google.protobuf.BoolValue](#google.protobuf.BoolValue) |  |  |
+| version | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="proto.spinnaker.Repository"></a>
+
+### Repository
+Spinnaker plugin repository flags
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| id | [string](#string) |  |  |
+| url | [string](#string) |  |  |
```

Comment:
Description of fields is missing. It's taken from the comments in `.proto` files: https://github.com/spinnaker/kleat/blob/master/api/proto/security/trust_store.proto#L17

Reply:
I included the description for Plugins & Repositories in the `.proto`

Reply:
I'm totally agree on include the plugin compatibility validation in the `kleat generate`, but the compatibility validation make http calls to the plugin repository and kleat is suppose to not depend on external resources. That's why I didn't include it in the `kleat generate`.

But I've been defining a mechanisms to save the compatibility metadata locally and use that information to validate compatibility, in that way we may include it in the `kleat generate`. I am planning to open another PR with those changes.